### PR TITLE
Clarify channel ports

### PR DESCRIPTION
**Pull request by velzevur**

### CODE REVIEW

File: epoch/api/README.md [hunk 1]
```diff
@@ -31,6 +31,13 @@ The epoch node exposes the following APIs:
     * It is defined in the rest of this document;
     * It is meant to be exposed on the Internet;
     * Its TCP port is configurable.
+  * Range of external Channels noise endpoints. It consists of as many TCP
+    endpoints as needed.
+    * They are encrypted and authenticated using the Noise protocol;
```
Comment (lucafavatella) [inline epoch/api/README.md]: Where is it defined? Via Swagger schema? In the rest of this document?

File: epoch/api/README.md [hunk 2]
```diff
@@ -31,6 +31,13 @@ The epoch node exposes the following APIs:
     * It is defined in the rest of this document;
     * It is meant to be exposed on the Internet;
     * Its TCP port is configurable.
+  * Range of external Channels noise endpoints. It consists of as many TCP
```
Comment (lucafavatella) [inline epoch/api/README.md]: Which Noise public key is used by the listener?
Which Noise public key of the other party does the listener expect/validate?

Comment (velzevur) [reply epoch/api/README.md]: [Addressed](https://github.com/aeternity/protocol/pull/229/commits/a086114ae96dd9f500cbc6e03869d5ee3a2b0502#diff-8b16649bc0188ba5b35d5f92f6b8656aR40)

File: epoch/api/README.md [hunk 3]
```diff
@@ -31,6 +31,15 @@ The epoch node exposes the following APIs:
...
```
Comment (lucafavatella) [inline epoch/api/README.md]: Could you also please clarify https://github.com/aeternity/epoch/blob/master/docs/configuration.md#channels for next release?

File: epoch/api/README.md [hunk 4]
```diff
@@ -31,6 +31,15 @@ The epoch node exposes the following APIs:
     * It is defined in the rest of this document;
     * It is meant to be exposed on the Internet;
     * Its TCP port is configurable.
```
Comment (uwiger) [inline epoch/api/README.md]: Also, the listen IP address is configurable (default: soon 127.0.0.1, but currently 0.0.0.0, meaning that the node listens on all available IP interfaces.)

File: epoch/api/README.md (hunk 3 above)
Comment (lucafavatella) [inline epoch/api/README.md]: Where is the schema of its payload defined?

Comment (velzevur) [reply epoch/api/README.md]: Clarified [in epoch](https://github.com/aeternity/epoch/pull/1667)

Comment (lucafavatella) [reply epoch/api/README.md]: I see addressed (`/channels/OFF-CHAIN.md#messages`). Thanks.

**Comment by velzevur** [comment]
The fragmentation is already tracked as [state channels should be able to reuse listen sockets](https://www.pivotaltracker.com/story/show/161143845). I've created a PT [SC WebSocket Versioning](https://www.pivotaltracker.com/story/show/161288395)

**Comment by velzevur** [comment]
@dincho - this is to be changed by [state channels should be able to reuse listen sockets](https://www.pivotaltracker.com/story/show/161143845)

**Pull request merged**